GetStream/stream-chat-swift
# Set team signing to Stream.io for all targets

# What this PR Does
- Set the current code signing team ID to the rest of the targets apart from Sample.

# How to test this
- Just build and run the application on your developer account device to see if you can run it.
